src/buildCommon.js

```diff
@@ -596,6 +585,14 @@ const makeVerb = function(group: ParseNode, options: Options): string {
     return text;
 };
 
+const makeGlue = (measurement: Measurement, options: Options): domTree.span => {
```

Comment:
I think a comment here would be useful. The term glue might be confusing to people.